app/views/concerns/paginated_view.rb

```diff
@@ -66,7 +66,8 @@ def pagination_total_pages
   end
 
   def pagination_page_item_count
-    paginated_set.count
+    return paginated_set['itemsCount'] unless paginated_set['itemsCount'].nil?
```

Comment:
The `PaginatedView` concern should not need to know about the API response's use of `itemsCount`. If anything this would have belonged in a method override in the `Portal::Index` view.

However, all that's needed to fix pagination for all including views is to rewrite this method:
```ruby
def pagination_page_item_count
  paginated_set.docs.count
end
```